<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>571</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="8" column="0">
      <widget class="QDoubleSpinBox" name="vitesse">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="prefix">
        <string>vitesse : </string>
       </property>
       <property name="suffix">
        <string> m/s</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>U</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <widget class="QDoubleSpinBox" name="tension_a">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Bookman Old Style</family>
        </font>
       </property>
       <property name="prefix">
        <string>Tension d'alimentation  = </string>
       </property>
       <property name="suffix">
        <string> V</string>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Bookman Old Style</family>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Mouvement :</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>U invers√©</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="mesurer">
         <property name="font">
          <font>
           <family>Bookman Old Style</family>
          </font>
         </property>
         <property name="text">
          <string>Mesurer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stop">
         <property name="font">
          <font>
           <family>Bookman Old Style</family>
          </font>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="graphique">
         <property name="font">
          <font>
           <family>Bookman Old Style</family>
          </font>
         </property>
         <property name="text">
          <string>Graphique</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="enregistrer">
         <property name="font">
          <font>
           <family>Bookman Old Style</family>
          </font>
         </property>
         <property name="text">
          <string>Enregistrer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Fichier</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label2">
       <property name="font">
        <font>
         <family>Bookman Old Style</family>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Circulaire</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QDoubleSpinBox" name="tension_v">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Bookman Old Style</family>
        </font>
       </property>
       <property name="prefix">
        <string>Tension verticale = </string>
       </property>
       <property name="suffix">
        <string> V</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QDoubleSpinBox" name="time">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="prefix">
        <string>time = </string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QDoubleSpinBox" name="courant_v">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Bookman Old Style</family>
        </font>
       </property>
       <property name="prefix">
        <string>Courant vertical = </string>
       </property>
       <property name="suffix">
        <string> A</string>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Bookman Old Style</family>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Mesures : </string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>Infini</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QLCDNumber" name="temps"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
